### ISSUE TITLE
Add term for Fluidigm C1 platform

### ISSUE THREAD

**Issue by malloryfreeberg** [HumanCellAtlas/ontology#37]
For new term requests, please provide the following information:

## Preferred term label

Fluidigm C1

## Synonyms

N/A

## Textual definition

The Fluidigm C1 is a microfluidic platform for single-cell RNA-Seq and DNA-Seq library preparation. Cell suspensions are added to the integrated fluidic circuit which shuttles individual cells into microfluidic chambers which are flooded with reagents in order to carry out library construction in very small volumes. 
[citation](https://www.earlham.ac.uk/fluidigm-c1)

## Suggested parent term

[RNA assay](https://ontology.staging.data.humancellatlas.org/ontologies/efo/terms?iri=http%3A%2F%2Fwww.ebi.ac.uk%2Fefo%2FEFO_0001457)

## Use case

Some early scRNA-seq is done using the Fluidigm C1 platform.

## Attribution

N/A

**Comment by malloryfreeberg** [comment]
@daniwelter 

Please see the following Slack thread related to what "Fluidigm C1" is.

tl;dr: We will need to be able to put "Fluidigm C1" as a library preparation method, whether that means adding the term to EFO in a particular way or expanding the library preparation method ontology scope in a particular way.

**Comment by malloryfreeberg** [comment]
Highlights of the thread:

Mallory Freeberg
Ambrose, out of curiosity, what you you consider the library preparation method for the following:

“Isolated single cells were re-suspended at a density of 700 live cells/μl in DMEM with 5% fetal calf serum (FCS). Cells were stained for DNA content and viability by supplementing the re-suspension buffer with NucBlue live cell stain (Life technologies, following the manufacturer’s protocol) and propidium iodide 10 μg/ml final. Cells were captured on the C1 system (Fluidigm) and processed using the SMARTer chemistry (Clontech) according to the Fluidigm protocol. External RNA Controls Consortium (ERCC) RNA spike-in mix was added to the lysis buffer 1:4000. C1 integrated fluidic circuits (IFCs) were imaged prior to cell lysis to identify sites occupied by single viable cells for downstream analyses. cDNA samples were selected after analyzing the cell images and prepared for sequencing using the Nextera XT library prep kit (Illumina). Libraries were sequenced using either Illumina HiSeq2500 100 bp paired-end sequencing or Illumina HiSeq4000 75 bp paired-end sequencing.”


Ambrose Carr  [29 minutes ago]
That's classic Fluidigm C1 + ERCC spike-ins

Mallory Freeberg  [28 minutes ago]
So Fluidigm, not Smart-seq, for lib prep protocol?


Ambrose Carr  [27 minutes ago]
> So Fluidigm, not Smart-seq, for lib prep protocol?

You are identifying the difficulty here. You show me that prep that says `live/dead stain -> fluidigm + Smarter -> nextera -> PE` and I go "oh, fluidigm".

Ambrose Carr  [27 minutes ago]
conversely, if you told me someone used fluidigm c1, I'd guess that meant smarter + nextera.

Ambrose Carr  [26 minutes ago]
However if you really wanted to understand what someone did, you'd need to know that they reverse transcribed with smarter chemistry, used some polymerase (?) for second strand synthesis, then added adapters with the nextera transposase.

Ambrose Carr  [26 minutes ago]
"Fluidigm C1" is thus essentially the name of a protocol that covers all the above information

**Comment by daniwelter** [comment]
We now have Fluidigm C1-based dissociation (http://www.ebi.ac.uk/efo/EFO_0010048). I will added Fluidigm C1-based library preparation as another EFO term.

**Comment by daniwelter** [comment]
Fluidigm C1-based library preparation is now in EFO under the ID http://www.ebi.ac.uk/efo/EFO_0010058 and will be publicly available after the August release.

**Comment by mshadbolt** [comment]
@zoependlington just following up on how long it would take for this term to be added into the HCAO? Is there a release scheduled? thanks

**Comment by daniwelter** [comment]
@zoependlington is on annual leave until the start of September so I wouldn't anticipate a release until then. If it's a massive emergency, I may be able to do a release early next week - otherwise I'm sure Zoe will be able to do one as soon as she's back.

**Comment by mshadbolt** [comment]
@daniwelter no not urgent. I was more just curious about the process and how long it would normally take to get our ontology once it is available in the EFO

**Comment by daniwelter** [comment]
EFO can normally be released in the HCA environments as soon as it's been released and available in the public OLS (about 24hrs after release). It just needs someone available to do the release. I believe a specific schedule was under discussion - 3 days after EFO release if memory serves - but this clashed with Zoe's annual leave this month.

**Comment by mshadbolt** [comment]
Ok great thanks for the info Dani, we can discuss further with Zoe once she is back from vacation.

**Comment by zoependlington** [comment]
@mshadbolt The next EFO release is on Monday 16th September, so I will get the next HCAO release out next week (Thursday at the latest), please let me know if there's any problem with this! 😄
